### PR TITLE
Added Mars orbital calculations

### PR DESCRIPTION
**Pull request by pllim**
Added Mars orbital calculations.

Because who doesn't like being told that their calculations are correct?

### CODE REVIEW

File: astropy/analytic_functions/mars_orbital.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions related to orbital calculations for Mars."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+__all__ = ['plug_my_laptop_in']
```
Comment (bsipocz) [inline astropy/analytic_functions/mars_orbital.py]: I suggest to rename the function to ``plug_my_laptop_in_Pleiades`` in order to remove any ambiguity.

Comment (pllim) [inline astropy/analytic_functions/mars_orbital.py]: Good point. But that would make non-Pleiadas users sad. We should give other computing clusters a chance.

**Comment by eteq** [comment]
This is a very clever solution to the problem.  And also clearly a good use of NASA supercomputer resources. :+1:

**Comment by eteq** [comment]
(Also, just in case anyone isn't paying attention, this was posted on April 1 so shouldn't be merged)

**Comment by pllim** [comment]
> this was posted on April 1 so shouldn't be merged

Haha! Thanks, Captain Obvious. :wink: 

For serious, I was gonna close this tomorrow.

**Pull request closed without merging**